mne/preprocessing/maxwell.py

```diff
@@ -505,3 +537,107 @@ def _update_sss_info(raw, origin, int_order, ext_order, nsens):
         raw.info['proc_history'] = [proc_block]
 
     return raw
+
+
+def _read_fine_cal(info, fine_cal_fname):
+    """Read sensor locations and imbalances from fine calibration file."""
+
+    logger.info('Fine calibration file used in Maxwell Filter: ' +
+                fine_cal_fname)
+
+    # Read new sensor locations
+    with open(fine_cal_fname, 'r') as fid:
+        cal_chans = []
+        lines = fid.readlines()
+        lines = lines[::-1]
+        while len(lines) > 0:
+            line = lines.pop()
+            if line[0] != '#':
```

Comment:
can save indentation of the following by doing `if line[0] == '#': continue` if you want